components/asic/bm1368.c

```diff
@@ -120,66 +105,27 @@ void BM1368_set_version_mask(uint32_t version_mask)
     _send_BM1368(TYPE_CMD | GROUP_ALL | CMD_WRITE, version_cmd, 6, BM1368_SERIALTX_DEBUG);
 }
 
-void BM1368_send_hash_frequency(float target_freq) {
-    float max_diff = 0.001;
-    uint8_t freqbuf[6] = {0x00, 0x08, 0x40, 0xA0, 0x02, 0x41};
-    uint8_t postdiv_min = 255;
-    uint8_t postdiv2_min = 255;
-    float best_freq = 0;
-    uint8_t best_refdiv = 0, best_fbdiv = 0, best_postdiv1 = 0, best_postdiv2 = 0;
-    bool found = false;
-
-    for (uint8_t refdiv = 2; refdiv > 0; refdiv--) {
-        for (uint8_t postdiv1 = 7; postdiv1 > 0; postdiv1--) {
-            for (uint8_t postdiv2 = 7; postdiv2 > 0; postdiv2--) {
-                uint16_t fb_divider = round(target_freq / 25.0 * (refdiv * postdiv2 * postdiv1));
-                float newf = 25.0 * fb_divider / (refdiv * postdiv2 * postdiv1);
-
-                if (fb_divider >= 144 && fb_divider <= 235 &&
-                    fabs(target_freq - newf) < max_diff &&
-                    postdiv1 >= postdiv2 &&
```

Comment:
This was changed from postdiv2 < postdiv1 to allowing them to be the same. I think this was an unintended change. See BM1366.